_boards_cm/maxlinear-evaluation-board.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: board
```

Comment:
This one is a `card`, and right now functionality is 'maybe', and driver required is also 'maybe' :)

Reply:
Which card folder does it go in, @geerlingguy? I'll get it fixed now. Thanks for bumping it!

Reply:
It goes under https://pipci.jeffgeerling.com/#other-cards (other).

Reply:
@geerlingguy Fixed it! New commit is below.